lib/tdf3/src/tdf.ts

```diff
@@ -981,6 +982,10 @@ export async function sliceAndDecrypt({
   cryptoService: CryptoService;
   segmentIntegrityAlgorithm: IntegrityAlgorithm;
 }) {
+  const limit = pLimit(navigator.hardwareConcurrency || 4); // save fallback number
```

Comment:
We might want to make this customizable by the caller somehow? (See also: chunksInOneDownload)

Reply:
I dont think we do tbh. Ive benchmarked it tried multiple configurations. More plimit with hardwareConcurrency amount of workers, vice-verca, more both of them. What i got: 
1)  number of Workers more than amount of cores really didnt bosted performance
2) pLimit ether didnt had any effect ether were so big that gained small improvement but froze ui